src/main/java/me/jellysquid/mods/sodium/client/render/pipeline/BlockRenderer.java

```diff
@@ -42,6 +43,7 @@ public class BlockRenderer {
 
     private final boolean useAmbientOcclusion;
     private boolean useSeparateAo;
+    public static boolean forceSodiumSmoothLighting = SodiumClientMod.options().advanced.useSodiumSmoothLighting;
```

Comment:
This should not be static, so that it changes whenever the renderer is reloaded (F3+A).

Reply:
It's getting set by the setter of the option, so it should change dynamically(regardless of the renderer getting reloaded) whenever the user changes the setting.